app/views/messages/_message.html.erb

```diff
@@ -22,9 +22,7 @@
         <%= message.user? ? 'You' : message.conversation.assistant.name %>
       </div>
 
-      <div class="text-zinc-700 dark:text-white" data-role="content_text">
-        <%= message.content_text %>
-      </div>
+      <div class="text-zinc-700 dark:text-white whitespace-pre-wrap" data-role="content_text"><%= message.content_text %></div>
```

Comment:
In order to get paragraph breaks in Chat's response to format correctly, I added whitespace-pre-wrap...but that means this all needs to be on one line or it renders weirdly.

Reply:
It's probably worth adding an HTML comment so someone doesn't reformat it in the future